framework/logging.cpp

```diff
@@ -2295,6 +2302,9 @@ void YamlLogger::print_header(std::string_view cmdline, Duration test_duration,
     logging_printf(LOG_LEVEL_QUIET, "command-line: '%s'\n", cmdline.data());
     logging_printf(LOG_LEVEL_QUIET, "version: " PROGRAM_VERSION "\n");
     logging_printf(LOG_LEVEL_VERBOSE(1), "os: %s\n", os_info().c_str());
+#if __has_include(<openssl/opensslv.h>)
+    logging_printf(LOG_LEVEL_VERBOSE(1), "ssl: %s\n", OPENssl_VERSION_TEXT);
```

Comment:
I'd suggest adding to `os_info()` so it gets printed like the libc version on the line above.